### PR TITLE
API Designer Bug Fixes

### PR DESCRIPTION
**Pull request by DimalChandrasiri**

### CODE REVIEW

File: composer/packages/api-designer/src/api-designer/components/context/open-api-context.ts [hunk 1]
```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (nadeeshaan) [inline composer/packages/api-designer/src/api-designer/components/context/open-api-context.ts]: Change year

File: composer/packages/api-designer/src/api-designer/components/header/header.tsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import * as React from "react";
```
Comment (NatashaWso2) [inline composer/packages/api-designer/src/api-designer/components/header/header.tsx]: Shall we add license headers for all the newly added files?

File: composer/packages/api-designer/src/api-designer/components/context/open-api-context.ts (hunk 1 above)
Comment (DimalChandrasiri) [reply composer/packages/api-designer/src/api-designer/components/context/open-api-context.ts]: Fixed with commit 4d0cc36

File: composer/packages/api-designer/src/api-designer/components/header/header.tsx (hunk 2 above)
Comment (DimalChandrasiri) [reply composer/packages/api-designer/src/api-designer/components/header/header.tsx]: Fixed with commit 4d0cc36

**Pull request merged**